docs/src/pages/components/data-grid/components/CustomColumnMenu.tsx

```diff
@@ -0,0 +1,120 @@
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
See #1627

```suggestion
import { makeStyles } from '@material-ui/styles';
```

Reply:
Well it's in progress... So I will rebase or she will :D